alacritty/src/event.rs

```diff
@@ -297,6 +311,17 @@ impl ApplicationHandler<Event> for Processor {
                 if let Ok(config) = config::reload(path, &mut self.cli_options) {
                     self.config = Rc::new(config);
 
+                    // Restart config monitor if imports changed.
+                    if let Some(monitor) = self.config_monitor.take() {
```

Comment:
Initially I considered adding special handling for when the config monitor is currently `None`, but there's only two scenarios where this can be the case:
 - Live config reload is disabled
 - No config was ever loaded

In both cases we'll obviously never get a config reload, so I think it makes sense to not consider those scenarios since we can never reach a `ConfigReload` event handler without a config monitor.